source/install/cloud-changelog.md

```diff
@@ -4,12 +4,74 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Workspace Limits
+ - Mattermost is releasing a free-forever Cloud offering.
+   - Added a new API endpoint ``GET /api/v4/usage/posts`` that returns the current usage limits for messages.
+   - Team Limit changes now propagate automatically.
+   - Added a new endpoint ``DELETE /api/v4/teams/{team_id:[A-Za-z0-9]+}/except`` which archives all teams except for the ``team_id`` passed to the endpoint.
+   - A server configured to use the Cloud Freemium limits now guards against enabling more integrations than the configured limit.
+Users are restricted from creating teams when at or above the teams limit for their plan.
+   - Fixed API ``GET /api/v4/usage/posts`` to include user posts only.
+   - Improved performance of API ``GET /api/v4/usage/posts`` by enabling caching of posts count.
+   - Users will be prevented from unarchiving teams if they're at or over the teams limit for their plan.
```

Comment:
Most of these we can omit because they are backend implementation details not relevant for Cloud. @jasonblais wondering if we should just have a generic description here on Cloud Freemium being released and the workspace limits that should be known